unparser.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency('equalizer',     '~> 0.0.9')
   gem.add_dependency('diff-lcs',      '~> 1.2.5')
   gem.add_dependency('concord',       '~> 0.1.5')
-  gem.add_dependency('parser',        '~> 2.2.2')
+  gem.add_dependency('parser',        '>= 2.2.2', '< 2.4')
```

Comment:
Its not possible to do such a change blindly like this. There are new nodes being added. See: https://github.com/whitequark/parser/blob/master/CHANGELOG.md#v230pre3-2015-11-25

Until parser handles these we cannot declare compatibility with 2.3 syntax.